src/components/inline/DonationProjectCard/DonationProjectCard.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import DonationProjectCard from "./DonationProjectCard";
+import background from "./coralreef.jpg";
+
+export default {
+  title: "Inline/DonationProjectCard",
+  component: DonationProjectCard,
+};
+
+const Template = ({ title, subtitle }) => (
```

Comment:
const Template = (args) => (

Reply:
👍 Changed to (args) param.

Reply:
I think I'd actually prefer the way it was originally with the destructured object in the function arguments, since it makes it clearer which properties are expected to be in the argument and it removes a little bit of visual noise by eliminating the `args` variable in the entire `Template` body. Do you have a specific reason for preferring `(args)`, @davidagustin?

Reply:
👍 Change it back to a destructured object { title, subtitle }.

Reply:
It was in your button component @richardxia so I assumed that was the way you wanted it.

Reply:
> It was in an example component online @richardxia

OK, if you don't have a strong preference for the `args` parameter, then I think I'd prefer the destructured form.

Reply:
I think the destructured form would be a lot better because it does reduce visual noise and also destructuring is a recurring style to react so it is more consistent with the general code.